app/views/accounts/_webauthn_platform.html.erb

```diff
@@ -20,12 +20,10 @@
     <div class="clearfix"></div>
   <% end %>
 </div>
-<% if IdentityConfig.store.platform_auth_set_up_enabled %>
-  <%= render ButtonComponent.new(
-        action: ->(**tag_options, &block) do
-          link_to(webauthn_setup_path(platform: true), **tag_options, &block)
-        end,
-        icon: :add,
-        class: 'usa-button usa-button--outline margin-top-2',
-      ).with_content(t('account.index.webauthn_platform_add')) %>
-<% end %>
\ No newline at end of file
+<%= render ButtonComponent.new(
+      action: ->(**tag_options, &block) do
+        link_to(webauthn_setup_path(platform: true), **tag_options, &block)
+      end,
+      icon: :add,
+      class: 'usa-button usa-button--outline margin-top-2',
```

Comment:
Not related, but since we're already here, might be good to use some of the `ButtonComponent` built-in features instead of the hardcoded classes.

```suggestion
      outline: true,
      class: 'margin-top-2',
```